artifactory/services/upload.go

```diff
@@ -775,6 +775,12 @@ func (us *UploadService) addFileToZip(artifact *clientutils.Artifact, progressPr
 		_, e = writer.Write([]byte(filepath.ToSlash(artifact.SymlinkTargetPath)))
 		return
 	}
+	// Check if directory
+	// Write directory to writer - path needs to end with a separator.
```

Comment:
// Check if directory
// Write directory to writer - path needs to end with a separator.
-->
// If this is a directory, add it to the writer with a trailing slash.